src/elastic-search.js

```diff
@@ -106,38 +106,52 @@ module.exports = function sendTestResults(testResultsLog, done) {
       });
     }
 
-    var hostname = repoConfig.elasticSearchHost
-    if (!!elasticsearchUsername === true || !!elasticsearchPassword === true) {
-      auth = `${elasticsearchUsername}:${elasticsearchPassword}@`
-      // Override any username:password in the elasticSearchHost param
-      hostname = `${auth}${hostname}`
+
+    var hosts = [];
+    const elasticHost = repoConfig.elasticSearchHost;
+
+    if (typeof elasticHost === 'string') {
```

Comment:
Correct, if you are writing to one instance, it will be backwards compatible.  

However, currently you can have the following:  

elasticSearchHost: ["elk1","elk2"] 

This `load balances` the data written across the two instances (elk1 and elk2).  So if you poll over the two instances, you get the entire record set, however, neither instance will contain all the records.  You have to poll both instances to get the entire record set.  

With this PR, this will need to be changed to: 

elasticSearchHost: [ ["elk:9200","elk:9201"] ]

This is the breaking change.  Let me know if that clears it up.

Reply:
makes sense. Is this something you think you would need for an elongated period of time or is this more of a migration plan needs?
If its a long term thing then we can merge this and i can publish a newer package for this. If this is just temporary then I suggest we should have a forked version and push that into your internal npm and use that. 
Either of the options work for me. Just let me know.

Reply:
We already have this forked and it's a temporary thing.  I can close the PR and if there is an ask for it later, we can reopen.  Thanks!